src/test/java/com/somemore/community/repository/CommunityCommentRepositoryTest.java

```diff
@@ -0,0 +1,62 @@
+package com.somemore.community.repository;
+
+import com.somemore.IntegrationTestSupport;
+import com.somemore.community.domain.CommunityComment;
+import com.somemore.community.repository.comment.CommunityCommentRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Transactional
+class CommunityCommentRepositoryTest extends IntegrationTestSupport {
+    @Autowired
+    CommunityCommentRepository communityCommentRepository;
+
+    @DisplayName("커뮤니티 게시글에 댓글을 생성할 수 있다. (Repository)")
+    @Test
+    void createCommunityComment() {
+
+        //given
+        UUID writerId = UUID.randomUUID();
+
+        CommunityComment communityComment = CommunityComment.builder()
+                .writerId(writerId)
+                .content("커뮤니티 댓글 테스트 내용")
+                .parentCommentId(null)
+                .build();
+
+        //when
+        CommunityComment savedComment = communityCommentRepository.save(communityComment);
+
+        //then
+        assertThat(savedComment.getWriterId()).isEqualTo(writerId);
+        assertThat(savedComment.getContent()).isEqualTo("커뮤니티 댓글 테스트 내용");
+        assertThat(savedComment.getParentCommentId()).isNull();
+    }
+
+    @DisplayName("커뮤니티 게시글에 대댓글을 생성할 수 있다. (Repository)")
```

Comment:
```suggestion
    @DisplayName("댓글에 대댓글을 생성할 수 있다. (Repository)")
```
이런 맥락이 더 좋지 않을까요?
그런 의미에서 parent가 없는 경우에 대한 검증이 여기서나 서비스 테스트에서 필요할 것 같아요!

Reply:
테스트 네임은 바꾸겠습니다!

부모 댓글 검증을 조회에서만 생각했네요 
추가하겠습니다! 감사합니다

Reply:
넵!

> board와 comment 파일이 헷갈릴 수 있겠다고 판단하여 폴더로 한 번 더 나누었는데 괜찮은지 여쭤봅니다.

이 부분도 저는 좋은 것 같습니다~

Reply:
재중님 생각해봤는데
댓글 생성 쪽에서는 삭제된 댓글에는 대댓글 생성을 못하도록 하고
댓글 조회 쪽에서 삭제된 경우에도 대댓글이 있으면 조회를 하도록 해야할 거 같은데
혹시 더 생각해야할 부분이 있을까요?